code/implementation/simulator/agent-connection/src/main/java/io/cattle/platform/agent/connection/simulator/impl/SimulatorInstanceInspectProcessor.java

```diff
@@ -0,0 +1,25 @@
+package io.cattle.platform.agent.connection.simulator.impl;
+
+import io.cattle.platform.agent.connection.simulator.AgentConnectionSimulator;
+import io.cattle.platform.agent.connection.simulator.AgentSimulatorEventProcessor;
+import io.cattle.platform.eventing.model.Event;
+import io.cattle.platform.eventing.model.EventVO;
+import io.cattle.platform.util.type.CollectionUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SimulatorInstanceInspectProcessor implements AgentSimulatorEventProcessor {
+
+    @Override
+    public Event handle(AgentConnectionSimulator simulator, Event event) throws Exception {
+        if (!"compute.instance.inspect".equals(event.getName())) {
```

Comment:
The reason I didn't was because it would introduce a new inter-project dependency. Will revisit to see if there is a cleaner way.